graph-builder/tests/mod.rs

```diff
@@ -4,3 +4,12 @@ extern crate failure;
 
 #[cfg(feature = "test-net")]
 mod net;
+
+#[cfg(feature = "test-e2e")]
+#[macro_use]
+extern crate assert_json_diff;
```

Comment:
This is not needed. Instead you can write `use assert_json_diff::name_of_the_macro` in the mod where the macro should be used.

Reply:
Fixed